### DISCUSSION TITLE
No Parking

### DISCUSSION THREAD

**Discussion by GStar5000** [ST-Apps/CS2-ExtendedRoadUpgrades#36] [Ideas]
First of all thanks for creating this mod. The one thing that would make it complete is to add the ability to enable No parking on individual sides of the road, similar to method of adding a Quay on each side of the road. I think this feature would be great for this Mod since the mod already adds additional Road Services and No Parking is the only road service missing.

**Comment by CitiesTest** [comment]
Something like this
If you have a four lane divided road and upgrade it with tram tracks they go in the middle and you only have one lane left on each side AND parking lots. So if you then could upgrade it so it removed the parking lots and have a lanes instead it would be grate.

Another Example is with any road that have parking lots you would then upgrade it with lanes and you would then for example have the tow lane road with 4 lanes for the same amount of units wide.

Obvious lanes are wider then the parking lots but it could be supplemented by making the sidewalk a little bit smaller.

because i think that is a very nice upgrade to the mod that would be lovely to se int the game.
